OpenRA.Mods.Common/Traits/Render/WithIdleOverlay.cs

```diff
@@ -35,6 +35,8 @@ public class WithIdleOverlayInfo : PausableConditionalTraitInfo, IRenderActorPre
 		[Desc("Custom palette is a player palette BaseName")]
 		public readonly bool IsPlayerPalette = false;
 
+		public readonly bool RenderIfBuildNotComplete = false;
```

Comment:
Please add a
`// TODO: Remove this when the buildComplete code is replaced with conditions.`
comment above this, just in case.

Reply:
Is it me, or is `RenderWhileBuilding` the same, but a lot cleaner?